src/test/util/setup_common.cpp

```diff
@@ -79,24 +73,6 @@ const std::function<std::string(const char*)> G_TRANSLATION_FUN = nullptr;
 /** Random context to get unique temp data dirs. Separate from m_rng, which can be seeded from a const env var */
 static FastRandomContext g_rng_temp_path;
 
-std::ostream& operator<<(std::ostream& os, const arith_uint256& num)
```

Comment:
was specifically requested to avoid repetition and since they're not mission-critical code: https://github.com/bitcoin/bitcoin/pull/30618#discussion_r1739923996

Reply:
Pushed some changes, let me know if it's any better, or if you insist on placing them in the cpp file in a different place than the header (with some explanations)

Reply:
I don't insist on placing them in a different place than in the header file, I just don't think this is worth the noise and bigger diff - I don't think anyone uses a declaration's location in the header to find the implementation in the cpp file?

Reply:
I think it makes it slightly more aligned with the header file, while the diff isn't a lot heavier.
What do you think @hodlinator?

Reply:
Was under the impression that Git had some smarts when it came to moving lines around, but at least naive `git blame` usage shows your name & commit on all the moved lines in the **.cpp**. It's straightforward code (going back in history is less necessary), and I understand your attempt to mirror the ordering from the **.h** file.

I think the ordering I pushed for in the **.h** file should stay either way, but maybe go with the more conservative approach in the **.cpp** as a compromise.

Reply:
No I don't feel strongly about it, I just think this change is not nearly useful enough to warrant an extra 33LoC diff. Anyway, not worth arguing over, like I said it's not blocking.